### ISSUE TITLE
[CI] MixedClusterClientYamlTestSuiteIT 370_doc_count_field/Test filters agg with doc_count

### ISSUE THREAD

**Issue by henningandersen** [elastic/elasticsearch#69638]
**Build scan**:
https://gradle-enterprise.elastic.co/s/dykbklriduuie
https://gradle-enterprise.elastic.co/s/d7mxuheigaghi (yaml rest compat)

**Repro line**:

```
/gradlew ':qa:mixed-cluster:v7.13.0#mixedClusterTest' \
  -Dtests.class="org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT" \
  -Dtests.method="test {p0=search.aggregation/370_doc_count_field/Test filters agg with doc_count}" \
  -Dtests.seed=2674840AC3F77C90 \
  -Dtests.security.manager=true \
  -Dtests.bwc=true \
  -Dtests.locale=es-BO \
  -Dtests.timezone=America/Kentucky/Monticello \
  -Druntime.java=11
```
**Reproduces locally?**:
No

**Applicable branches**:

master at least

**Failure history**:

Started failing Feb 24, related to #68930. [Build-stats](https://build-stats.elastic.co/app/kibana#/discover?_g=(refreshInterval:(pause:!t,value:0),time:(from:now-30d,mode:quick,to:now))&_a=(columns:!(branch),index:e58bf320-7efd-11e8-bf69-63c8ef516157,interval:auto,query:(language:lucene,query:'%22370_doc_count_field%2FTest%20filters%20agg%20with%20doc_count%22'),sort:!(time,desc)))
**Failure excerpt**:

```
Failure at [search.aggregation/370_doc_count_field:181]: value of [profile.shards.0.aggregations.0.debug.segments_with_doc_count_field] is not comparable (got [null])	
```

**Comment by nik9000** [comment]
I believe I pushed a commit to disable this test in mixed clusters. I'll
double check in a moment.

**Comment by nik9000** [comment]
Closed by 19eef6a258dd7319d27794860ab46550141e1e4d.
